app/Http/Controllers/Api/Admin/SensorHistoryController.php

```diff
@@ -204,83 +205,63 @@ public function fetchAndStoreSensorData($eui)
 
             $url = "https://api-urbantree.alumnat.iesmontsia.org/sensors/deveui/{$eui}/history";
             if ($lastFetchDate) {
-                $url .= '?last_fetch_date='.urlencode($lastFetchDate);
+                $url .= '?last_fetch_date=' . urlencode($lastFetchDate);
```

Comment:
S'hauria d'executar `/vendor/bin/pint`

```suggestion
                $url .= '?last_fetch_date='.urlencode($lastFetchDate);
```